pack/tcu/icc_encounter.json

```diff
@@ -0,0 +1,563 @@
+[
+	{
+		"back_text": "Hard / Expert\n[skull]: -X. X is 1 higher than the total amount of doom and breaches on your location.\n[cultist]: Reveal another token. If there are fewer than 3 breaches on your location, place 1 breach on your location.\n[tablet]: -3. For each point you fail by, remove 1 breach from the current act.\n[elder_thing]: -4. If you fail, place 1 breach on a random location.",
+		"code": "05284",
+		"double_sided": true,
+		"encounter_code": "in_the_clutches_of_chaos",
+		"encounter_position": 1,
+		"faction_code": "mythos",
+		"name": "In the Clutches of Chaos",
+		"pack_code": "icc",
+		"position": 284,
+		"quantity": 1,
+		"text": "Easy / Standard\n[skull]: -X. X is the total amount of doom and breaches on your location.\n[cultist]: Reveal another token. If there are fewer than 3 breaches on your location, place 1 breach on your location.\n[tablet]: -2. For each point you fail by, remove 1 breach from the current act.\n[elder_thing]: -3. If you fail, place 1 breach on a random location.",
+		"type_code": "scenario"
+	},
+	{
+		"back_flavor": "The stars vanish in the night sky as the maw of the void swallows the horizon. The muffled beating of maddening drums and the thin whine of otherworldly flutes drift through the emptiness.",
+		"back_name": "The Final Incantation",
+		"back_text": "Check Campaign Log.\n- <i>If Anette Mason is possessed by evil, <b>(→R3)</b>.\n- <i>If Carl Sanford possesses the secrets of the universe,</i><b>(→R4)</b>.",
```

Comment:
Please add closing italics markers after 'evil,' and add a space after 'universe,'.